dev_scripts/env.py

```diff
@@ -70,6 +70,10 @@
         | apt-key add -
 """
 
+DOCKERFILE_UBUNTU_2304_REM_USER = r"""
+RUN touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu
```

Comment:
One small request here, if you can please link to either a GitHub issue of ours, or an external link, to explain why this exception is necessary.

Reply:
Sure. I had added it in the commit message but I agree that it's best left in the code. Fixed in https://github.com/freedomofpress/dangerzone/pull/461/commits/8dbb2fe9e04f76c793960ec47989dd0748840fea.